package.json

```diff
@@ -27,7 +27,7 @@
   },
   "homepage": "https://github.com/balena-io/docs",
   "dependencies": {
-    "@resin.io/doxx": "0.11.3",
+    "@resin.io/doxx": "0.11.4",
```

Comment:
Should this now be balena.io instead of resin.io? :eyes:
:man_shrugging:

Reply:
That upstream repo hasn't seen much attention in a while. It is still in the resin namespace - there is an open PR on this https://github.com/balena-io/doxx/pull/28